src/dotty/tools/dotc/reporting/Reporter.scala

```diff
@@ -206,6 +230,10 @@ abstract class Reporter extends interfaces.ReporterResult {
    */
   def errorsReported = hasErrors
 
+  private[this] var reportedFeaturesUseSites = Set[Symbol]()
```

Comment:
Currently it does not suppress the warning. It is only there to make sure the explanation of how to avoid it is only printed once per site.